### ISSUE TITLE
Why doesn't the app receive messages push notifications after version 2.14.0?

### ISSUE THREAD

**Issue by stilleshan** [chatwoot/chatwoot#6753]
It only pushes notifications when a message is created, and no notifications will be pushed for each subsequent message.

**Comment by ankitseal** [comment]
Any way to rollback to 2.14.0? Maybe using cwctl?

**Comment by stilleshan** [comment]
> Any way to rollback to 2.14.0? Maybe using cwctl?

I was able to receive messages notification normally after switching back to v2.13.1 docker images

**Comment by ankitseal** [comment]
> > Any way to rollback to 2.14.0? Maybe using cwctl?
> 
> I was able to receive messages notification normally after switching back to v2.13.1 docker images

I have it installed on bare metal. I don't notifications even in the browser. I will have to try and rollback and wait for v3 before I upgrade.

**Comment by vishnu-narayanan** [comment]
@stilleshan Could you provide more details?

**Describe the bug**

A clear and concise description of what the bug is.

**To Reproduce**

Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See the error

**Expected behavior**

Share a clear and concise description of what you expected to happen.

**Screenshots**

If applicable, add screenshots to help explain your problem.

**Browser logs**

Share the browser logs to debug the issue further.

**Server logs**

Share the server logs to debug the issue further.

**Environment**

Describe whether you are using Chatwoot Cloud (app.chatwoot.com) or a self-hosted installation of Chatwoot. If you are using a self-hosted installation of Chatwoot, describe the type of deployment (Docker/Linux VM installation/Heroku/Kubernetes/Other).

- [ ] app.chatwoot.com (Chatwoot Cloud)
- [ ] Self-hosted
- - [ ] Linux VM
- - [ ] Docker
- - [ ] Kubernetes
- - [ ] Heroku
- - [ ] Other (Please specify)


**Desktop (please complete the following information)** (If applicable)
 - OS: [e.g. Linux, Windows, MacOS]
 - Browser [e.g. chrome, firefox, safari]
 - Version [e.g. 22]

**Smartphone (please complete the following information)** (If applicable)
 - Device: [e.g. iPhone6, Pixel7]
 - OS: [e.g. iOS8.1]
 - Browser [e.g. stock browser, firefox, safari]
 - Version [e.g. 22]

**Docker** (If applicable)

Please share the output of the following.
- `docker version`
- `docker info`
- `docker-compose version`

**Cloud Provider** (If applicable)
- [ ] AWS
- [ ] GCP
- [ ] Azure
- [ ] DigitalOcean
- [ ] Others

**Additional context**

Add any other context about the problem here.

**Comment by vishnu-narayanan** [comment]
@ankitseal 

> Any way to rollback to 2.14.0? Maybe using cwctl?

`cwctl` does not support rollbacks at the moment. You could manually switch to any release you want. You can follow the manual upgrade process and checkout to the tag you want. 

Ref: https://www.chatwoot.com/docs/self-hosted/deployment/linux-vm#upgrading-to-a-newer-version-of-chatwoot

**Comment by stilleshan** [comment]
CentOS 7.6
docker 20.10.22
docker-compose v2.14.2

I deploy chatwoot by docker-compose with chatwoot/chatwoot:laste first at 13 Feb, I checked docker hub, the latest version at that time should be v2.13.1.

I deployed the service exactly according to the official documentation, and configured the nginx reverse proxy.

everything was fine at that time.

I can receive push notifications like:
"[New conversation] - #12 has been created in XXXX"
"[New message] - #12 xxxx"

After a few days, i used watchtower update chatwoot/chatwoot to the latest version,I guess it should be v2.14.0.
At this point I didn't notice the push issue
A few days late, watchtower update it maybe v2.15.0?  I found i cann't update index settings. it's show 500 error.
At this time, I realized that I have not received message pushes in the past few days
**I JUST RECEIVE "[New conversation] - #12 has been created in XXXX"
I CAN NOT RECEIVE "[New message] - #12 xxxx"**

So i downgraded to v2.14.0. push problem still persists.
When i downgraded to v2.13.1 everything was fine.
I haven't touched any settings or databse these days. just modify images version in docker-compose.yml

**Comment by ankitseal** [comment]
I resolved my issue with notifications. Turns out it websockets were not able to connect with my Chatwoot. Updated my nginx reverse proxy using the following template. All working fine now!
https://github.com/chatwoot/chatwoot/blob/develop/deployment/nginx_chatwoot.conf

**Comment by vishnu-narayanan** [comment]
@stilleshan Could you check if you are having issues with web sockets?

**Comment by stilleshan** [comment]
Thank you everyone above, I don't think it's a configuration issue. As I mentioned, I followed the official documentation to deploy it since version v2.13.1 and correctly configured nginx websockets to receive any notifications normally.

The current situation is that message push is divided into two types, one is "New conversation" and the other is "New message". After I upgraded to v2.14, neither the web nor the iOS app received "New message" notifications, not that there are messages that are not pushed.

During this period, I did not modify any settings or configurations.
